client/src/cmdhw.c

```diff
@@ -743,6 +743,39 @@ static int CmdSetDivisor(const char *Cmd) {
     return PM3_SUCCESS;
 }
 
+static int CmdSetHFThreshold(const char *Cmd) {
+
+    CLIParserContext *ctx;
+    CLIParserInit(&ctx, "hw sethfthresh",
+                  "Set thresholds in HF/14a and Legic mode.",
+                  "hw sethfthresh -t 7 -i 20 -l 8"
+                 );
+
+    void *argtable[] = {
+        arg_param_begin,
+        arg_int0("t", "thresh", "<dec>", "threshold, used in 14a reader mode (def 7)"),
+        arg_int0("i", "high", "<dec>", "high threshold, used in 14a sniff mode (def 20)"),
+        arg_int0("l", "legic", "<dec>", "threshold used in Legic mode (def 8)"),
+        arg_param_end
+    };
+    CLIExecWithReturn(ctx, Cmd, argtable, true);
+    uint8_t params[3];
+    params[0] = arg_get_int_def(ctx, 1, 7);
+    params[1] = arg_get_int_def(ctx, 2, 20);
+    params[2] = arg_get_int_def(ctx, 3, 8);
+    CLIParserFree(ctx);
+
+    if ((params[0] < 1) || (params[0] > 63) || (params[1] < 1) || (params[1] > 63)) {
+        PrintAndLogEx(ERR, "Thresholds must be between " _YELLOW_("1") " and " _YELLOW_("63"));
+        return PM3_EINVARG;
+    }
+
+    clearCommandBuffer();
+    SendCommandNG(CMD_HF_ISO14443A_SET_THRESHOLDS, (uint8_t *)&params, sizeof(params));
```

Comment:
The preferred way is to use a struct.   
Makes things more easier to understand and follow.

Reply:
I'll change that.